evidence/alexggh/0001-2024-10-21-rank-1-retention.md

```diff
@@ -0,0 +1,61 @@
+# Evidence-0001: Retention at Rank I
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | 2024/10/21                                                                                  |
+| **Submitted by**| Alexandru Gheorghe                                                                          |
+
+
+## Member details
+
+- Matrix username: @alexggh:parity.io
+- Polkadot address: 14uA7Vc828e2Q6oL5GBHP9UzTkEvwqbroERwRmucGrLmPuuL
+- Current rank: 1
+- Date of initial induction: 2024/08/14
```

Comment:
Initial induction was: **2024/07/18**.

Reply:
This is my referenda for induction: https://collectives.subsquare.io/fellowship/referenda/175?tab=timeline.

That was executed on `2024-08-14 12:54:12` I considered that the date of initial induction, where did you take the `2024/07/18` ?

Reply:
I took the info from here: https://collectives.statescan.io/#/extrinsics/4243177-2 , but I misread a number: it should be 2024/07/20. 
The referendum executed was for your promotion to rank 1, which happened after the induction.
In any case, it looks odd if your report date is 25th July whereas the induction date is 18th August. It should be the other way round.

Reply:
> I took the info from here: https://collectives.statescan.io/#/extrinsics/4243177-2 , but I misread a number: it should be 2024/07/20.

Totally makes sense, I'll use that, thank you!